src/pages/AppDetail.svelte

```diff
@@ -19,49 +21,53 @@
   @include metadata-table;
 </style>
 
-{#await appDataPromise then app}
-  {#if app.prototype}
-    <AppAlert
-      status="warning"
-      message="This application is a prototype. The metrics and pings listed below may contain inconsistencies and testing strings." />
-  {/if}
-  <PageTitle text={params.app} />
+<div transition:fade>
```

Comment:
I'd suggest figuring out transitions in a seperate PR. I totally agree we should do them, but I'd like to approach them deliberately.